AssessmentA1/TaskManager/EmployeeManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace TaskManager
+{
+    public class EmployeeManager : List<Employee> 
+    { 
+        public List<Employee> ListOfEmployees { get; set; } = new List<Employee>();
```

Comment:
```suggestion
        public List<Employee> Employees { get; set; } = new List<Employee>();
```

Don't use the phrase such as "list" in the name of the variable.

Reply:
Please fix the same in other locations. I noticed you followed this naming convention in multiple places.

Reply:
Changed the variable names of the list.